### ISSUE TITLE
Multi Winner Percent Calculation

### ISSUE THREAD

**Issue by iambateman** [iambateman/RankIt#29]
Reference URL: https://rankit-vote.firebaseapp.com/results/ndaYgvDrHGsypjbjBROy/summary

In some cases, in multi-winner, I'm losing between 1-6% low. Here, on the summary, it's showing 20, 33, 43. In the first round it adds up to 99%.

Trying to figure out why that's happening and wanted to see if Owen had any insight into what I'm doing wrong. 

Each percentage is calculated individually, based on where we are in the round. The function is in `results.service.ts, line 13` 

If I had to guess, it would be that I'm making an assumption about past winners that is wrong. I'm going to look at it more this weekend to see if I can sort it out.

**Comment by ggordn3r** [comment]
Just to document this, we realized in our last meeting that this was caused by _exhausted votes_. Say someone only votes once for a candidate that is eliminated. Their vote will no longer be counted, so in later rounds the total ballots counted will shrink compared to the original.

Now, a question for @oburbank -- should we explain this at all? For instance, when the total does not add up to 100% we could add a sentence to the round summary explaining exhausted votes.
